{{ cookiecutter.project_slug }}/api/Makefile

```diff
@@ -1,24 +1,33 @@
 .ONESHELL:
 
+poetry-install:
```

Comment:
why not use just `install` here, in other cases with `lock` and `shell`, the context can be different but here I think we can be simple.

Reply:
Make sense. And if we replace `poetry` theme of command wouldn't change.